src/core/util/tools.py

```
@@ -23,18 +23,19 @@
 from src.core.constants import Constants
 
 
-def run_shell(shell: str) -> str:
+def run_shell(shell: str, log_ignore_regex: str = None) -> str:
     Constants.log.info(f"[shell] {shell}")
     cmd = subprocess.Popen(shell, stdin=subprocess.PIPE, stderr=subprocess.PIPE, stdout=subprocess.PIPE,
```

Ruff (0.13.1)

`subprocess` call with `shell=True` identified, security issue

(S602)